content/en/docs/concepts/storage/storage-capacity.md

```diff
@@ -0,0 +1,116 @@
+---
+reviewers:
+- jsafrane
+- saad-ali
+- msau42
+- xing-yang
+- pohly
+title: Storage Capacity
+content_type: concept
+weight: 45
+---
+
+<!-- overview -->
+
+Storage capacity is limited and may vary depending on the node on
+which a pod runs: network-attached storage might not be accessible by
+all nodes, or storage is local to a node to begin with.
+
+This page describes how Kubernetes keeps track of storage capacity and
+how the scheduler uses that information to schedule Pods onto nodes
+that have access to enough storage capacity for the remaining missing
+volumes. Without storage capacity tracking, it is random whether the
+selected node can run the Pod and multiple scheduling retries may be
+needed.
+
+<!-- body -->
+
+
+## Enabling the feature
+
+Storage capacity tracking is an *alpha feature* and only enabled when
+the `CSIStorageCapacity` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is enabled. A quick check
+whether a Kubernetes cluster supports the feature is to list
+`CSIStorageCapacity` objects with:
+```shell
+kubectl get csistoragecapacities --all-namespaces
+```
+
+If supported, the response will a list of objects or:
+```
+No resources found
+```
+
+If not supported, this error is printed instead:
+```
+error: the server doesn't have a resource type "csistoragecapacities"
+```
+
+In addition to enabling the feature in the cluster, a [CSI
+driver](/docs/concepts/storage/volumes/#csi) deployment also has to
+support it. Please refer to the driver's documentation for
+details. The feature is not supported for non-CSI storage systems.
+
+Without this support, there will be no information about storage
+capacity available through the driver and the scheduler will schedule
+Pods with volumes provided by the driver without looking for capacity
+information.
```

Comment:
This could move to the page introduction, I think.

Reply:
I already added something there in response to @jsafrane's comment ("Without storage capacity tracking..."). I removed it here.